include/spdk/bdev.h

```diff
@@ -229,6 +230,8 @@ struct spdk_bdev_io *spdk_bdev_unmap(struct spdk_bdev *bdev, struct spdk_io_chan
 struct spdk_bdev_io *spdk_bdev_flush(struct spdk_bdev *bdev, struct spdk_io_channel *ch,
 				     uint64_t offset, uint64_t length,
 				     spdk_bdev_io_completion_cb cb, void *cb_arg);
+struct spdk_bdev_io *spdk_bdev_passthru(struct spdk_bdev *bdev, void *cmd, void *buf,
```

Comment:
This should be named spdk_bdev_nvme_passthru() to match the I/O type, as mentioned above (and if we decide to make admin and I/O two separate I/O types, also include that as part of the passthru function name).

cmd should be declared as `const struct spdk_nvme_cmd *`.